docs/conf.py

```diff
@@ -50,7 +50,7 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = ['_build', 'cloud.rst', 'core_graph.rst', 'geo_types.rst', 'graph.rst', 'graph_fluent.rst']
```

Comment:
we removed those deliberately...
since it's not relevant to scylla 

@tzach maybe we should just clear the link to those ? 

@dgarcia360 any way to mark those warning acceptable ? like linter ignore comments ? or anything like that ?

Reply:
No problem, what we could do is to remove this sentence from the docs and exclude both files: https://github.com/scylladb/python-driver/pull/77/files#diff-d9b149498982c0663c3b7170398773361ed5678f1a627e9c2fd8d2c955c563db

Reply:
@fruch Reverted the changes in https://github.com/scylladb/python-driver/pull/77/commits/c3fdd5820303c3bdfebf52e9d70ba9edbaa23038 and deleted the sentence raising the warning.